packages/expo-face-detector/src/FaceDetector.ts

```diff
@@ -0,0 +1,61 @@
+import { UnavailabilityError } from 'expo-errors';
+
+import ExpoFaceDetector from './ExpoFaceDetector';
+
+export type Point = { x: number; y: number };
```

Comment:
Prefer to keep this internal (remove `export`) since it's so generically named and several different modules could all export a `Point` type, plus people probably don't need to refer to this type directly.